examples/jsm/nodes/inputs/CubeTextureNode.js

```diff
@@ -36,7 +36,19 @@ CubeTextureNode.prototype.generate = function ( builder, output ) {
 	}
 
 	var cubetex = this.getTexture( builder, output );
-	var uv = this.uv.build( builder, 'v3' );
+
+	
+	var uv = this.uv || builder.context.cubeUv || new ReflectNode();
+
+	if ( !uv ){
```

Comment:
I think that here uv never will be undefined...

Reply:
I removed the default value from the constructor. This allows uv to be contextually defined here.  If there is no contextual uv, the default value is still `new ReflectNode()`.